src/components/shared/__tests__/DocumentTitle.js

```diff
@@ -0,0 +1,61 @@
+import DocumentTitle from 'shared/DocumentTitle';
+import React from 'react';
+import { getComponentWithTheme } from 'testUtils/renderUtils';
+import { render } from '@testing-library/react';
+
+let initialTitle = '';
+
+const getComponent = props => {
+  const propsWithChildren = Object.assign({}, props, { children: <p /> });
+
+  return getComponentWithTheme(DocumentTitle, propsWithChildren);
+};
+const renderWithProps = props => render(getComponent(props));
+
+describe('DocumentTitle', () => {
+  beforeAll(() => {
+    initialTitle = document.title;
+  });
+
+  afterAll(() => {
+    document.title = initialTitle;
+  });
+
+  beforeEach(() => {
+    document.title = initialTitle;
+  });
```

Comment:
nitpick 1: I like to follow a chronological order here, it helps me in understanding what's going on